examples/cloudflare-workers/src/worker.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * Welcome to Cloudflare Workers! This is your first worker.
+ *
+ * - Run `npm run dev` in your terminal to start a development server
+ * - Open a browser tab at http://localhost:8787/ to see your worker in action
+ * - Run `npm run deploy` to publish your worker
+ *
+ * Learn more at https://developers.cloudflare.com/workers/
+ */
+
+class MomentoFetcher {
+	private readonly apiToken: string;
+	private readonly baseurl: string;
+	constructor(token: string, endpoint: string) {
+		this.apiToken = token;
+		this.baseurl = `https://api.cache.${endpoint}/cache`;
+	}
+
+	async get(cacheName: string, key: string) {
+		const resp = await fetch(`${this.baseurl}/${cacheName}?key=${key}&auth_token=${this.apiToken}`);
+		if (resp.status < 300) {
+			console.log("successfully retrieved item from cache")
+		} else {
+			throw new Error("failed to retrieve item from cache")
+		}
+		
+		return await resp.text();
+	}
+
+	async set(cacheName: string, key: string, value: string, ttl_milliseconds: number = 3000) {
+		const resp = await fetch(`${this.baseurl}/${cacheName}?key=${key}&auth_token=${this.apiToken}&&ttl_milliseconds=${ttl_milliseconds}`, {
+			method: 'PUT',
+			body: value
+		});
+
+		if (resp.status < 300) {
+			console.log(`successfully set ${key} into cache`);
+		} else {
+			throw new Error(`failed to set item into cache message: ${resp.statusText} status: ${resp.status}`);
+		}
+
+		return;
+	}
+
+	async delete(cacheName: string, key: string) {
+		const resp = await fetch(`${this.baseurl}/${cacheName}?key=${key}&auth_token=${this.apiToken}`, {
+			method: 'DELETE',
+		});
+		if (resp.status < 300) {
+			console.log(`successfully deleted ${key} from cache`);
+		} else {
+			throw new Error(`failed to delete ${key} from cache. Message: ${resp.statusText}; Status: ${resp.status}`);
+		}
+
+		return resp;
+	}
+}
+
+export interface Env {
+	MOMENTO_AUTH_TOKEN: string;
+	MOMENTO_REST_ENDPOINT: string;
```

Comment:
If we can infer the endpoint through the token, can we just infer it programmatically in this example? Like I am not sure how beneficial is to copy from the console if they're creating applications where they hard code it anyway. Console is helpful for ```curl``` and other-use cases though